README.md

```diff
@@ -70,6 +70,32 @@ const puppeteer = require('puppeteer');
 
 See [`Page.pdf()`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagepdfoptions) for more information about creating pdfs.
 
+**Example** - evaluate script in the context of the page
+
+```js
+const puppeteer = require('puppeteer');
+
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  await page.goto('https://example.com');
+  
+  // Get the viewport of the page.
+  const viewport = await page.evaluate(() => {
```

Comment:
I'd rename s/viewport/dimensions to avoid confusion with `page.viewport()` method